src/App.css

```diff
@@ -1,39 +1,77 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
```

Comment:
- [ ] **Design**:
- In reading the project requirement, I would advise you to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
The Layout: **composition and space between elements**.
Typography: font face, size, and weight.

- [ ] See your design: **Home page**
![Capture](https://user-images.githubusercontent.com/63972670/225519214-46d31730-15a9-4292-b28e-206e17ddba49.PNG)

- [ ] The Original design: **Home and Detail**


![Capture1](https://user-images.githubusercontent.com/63972670/225519577-5780b8ed-05e3-454f-8974-b31507564840.PNG)